src/assets/svg-components/MassaLogo.tsx

```diff
@@ -0,0 +1,35 @@
+export function MassaLogo() {
+  return (
+    <svg
+      width="24"
```

Comment:
Can't we have size as params to change it here if needed!? What if we need the logo in 50px!? 🤔

Reply:
has been added !